linux_sql/README.md

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
README is too short and no detail. Please see the sample README attached in the Notion ticket